scylla/src/response/query_result.rs

```diff
@@ -207,6 +207,8 @@ impl QueryResult {
     /// Ok(())
     /// # }
     /// ```
+    // `allow(clippy::result_large_err)` is fine. `QueryRowsResult` is smaller than `IntoRowsResultError`.
+    #[allow(clippy::result_large_err)]
```

Comment:
:question: Didn't you mean: `is larger`?

Reply:
Otherwise the explanation is counter-convincing.

Reply:
Right, sorry.